### PR TITLE
Update compat to CxxWrap 0.16

### LINKED ISSUE #62: Use newer version of CxxWrap
Currently, SDPA fails on PkgEval:

[PkgEval run](https://juliaci.github.io/NanosoldierReports/pkgeval_badges/S/SDPA.html)

The culprit seems to be CxxWrap. However, CxxWrap alone runs fine:

[PkgEval run](https://juliaci.github.io/NanosoldierReports/pkgeval_badges/C/CxxWrap.html)

I think simply supporting a newer version (v0.16 or maybe even only v0.15) would solve that.
On this note, I suggest to [reactivate CompatHelper](https://github.com/jump-dev/SDPA.jl/actions/workflows/CompatHelper.yml).

### PR DESCRIPTION
**Pull request by odow**
Closes #62

### CODE REVIEW

**Comment by odow** [comment]
Requires https://github.com/JuliaRegistries/General/pull/118250

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: ['1.6', '1']  # Test against LTS
+        version: ['1.10', '1']  # Test against LTS
```
Comment (schillic) [inline .github/workflows/ci.yml:18]: The new SDPA version does not work on v1.6 but v1.6 is still marked as compatible in `Project.toml`. So now the new SDPA version installs in v1.6 and breaks.
(`UndefVarError: get_libsdpawrap_path not defined`)

Comment (blegat) [reply .github/workflows/ci.yml:18]: I thought I bumped it but I went to fast, let's update it on the registry

Comment (blegat) [reply .github/workflows/ci.yml:18]: Should be fixed by https://github.com/JuliaRegistries/General/pull/119139

**Pull request merged**